pkg/ibmvpc/examples/out/analysis_out/acl_testing3_all_vpcs__debug.txt

```diff
@@ -4431,9 +4431,11 @@ vsi3c-ky[10.240.30.4] => vsi3b-ky[10.240.30.6] : All Connections
 64.0.0.0/3 => vsi2-ky[10.240.20.4] : All Connections
 8.0.0.0/7 => vsi2-ky[10.240.20.4] : All Connections
 96.0.0.0/6 => vsi2-ky[10.240.20.4] : All Connections
+db-endpoint-gateway-ky[10.240.30.7] => vsi1-ky[10.240.10.4] : protocol: UDP,ICMP
```

Comment:
How is it that we got new connections?

Reply:
This is in the `debug` output format, under `stateful combined connections`.
The connections added are those for which stateful analysis does not apply, thus they are also included in this output now.  (this category in the debug output represents those connections for which "unidirectional" should not be considered).

Reply:
got it. thanks.